### ISSUE TITLE
Getting Azure Code signing

### ISSUE THREAD

**Issue by trhumphries** [Azure/azure-code-signing-action#11]
There's no info on getting an Azure Code signing certificate.

**Comment by veikkoeeva** [comment]
https://twitter.com/filipnavara/status/1656401554390282242

> You don't hold the private key but you have control over it. You send hashes to the service and it signs them with your cert (issued by Azure; trusted by Windows). There's integration with SignTool, PowerShell, MsBuild, Azure Pipelines,...

But looking at that discussion, maybe it needs to be clarified here too, somehow.

**Comment by rednoah** [comment]
Keep us posted.

**Comment by Jaxelr** [comment]
Yes, I'll make sure to update this issue once we release more information to public preview

**Comment by MWGMorningwood** [comment]
I'm very eager to hear more about this!
With more and more developers entering the scene, fast and easy code signing will be a massive win.

**Comment by praneetloke** [comment]
@Jaxelr Is there a way to request to opt-in to the private preview?

**Comment by MWGMorningwood** [comment]
> @Jaxelr Is there a way to request to opt-in to the private preview?

I saw something that mentioned that organizations must have three years of history and be in good standing/trusted by MS to be a part of it. Beyond that, I'm very curious as well. It didn't mention a signup process or anything.

**Comment by ianjmcm** [comment]
Earlier in the preview we were only supporting organizations with 3 years of history, but we now can support all organizations and individuals. That said, we are no longer onboarding folks into the limited preview as we are heads down on the upcoming public launch.

**Comment by MWGMorningwood** [comment]
> Earlier in the preview we were only supporting organizations with 3 years of history, but we now can support all organizations and individuals. That said, we are no longer onboarding folks into the limited preview as we are heads down on the upcoming public launch.

Got it! Well, fingers crossed for a timely launch :)
I have faith!

**Comment by nazar-pc** [comment]
Any updates @ianjmcm? As an OSS developer with automated builds I'd really like to sign stuff in CI before uploading artifacts to release on GitHub with this action and hopefully don't pay a lot out of my own pocket for free software.

**Comment by Jaxelr** [comment]
hey for folks on this thread, as posted on this [PR](https://github.com/Azure/azure-code-signing-action/pull/26) this repo is getting renamed to [trusted signing action](https://github.com/azure/trusted-signing-action) and more relevant for this discussion, we expect an announcement in the coming weeks.

**Comment by praneetloke** [comment]
> more relevant for this discussion, we expect an announcement in the coming weeks.

That's awesome! I noticed the changes and was hoping that an announcement for a public preview at the very least is imminent. Really excited for this as I am looking to get my Pulumi desktop app signed for Windows very soon.

**Comment by praneetloke** [comment]
Is Trusted Signing still in preview phase? Because I see this:

> Trusted Signing at this time can only onboard Legal Business Entities that have verifiable tax history of three or more years.

https://learn.microsoft.com/en-us/azure/trusted-signing/quickstart?tabs=registerrp-portal%2Caccount-portal%2Ccertificateprofile-portal%2Cdeleteresources-portal#important-information-for-public-identity-validation

I can't seem to register the resource provider for CodeSigning using the steps outlined in the documentation (as in nothing happens when I click Register) and I believe it might have something to do with the above statement about tax history.

**Comment by Jaxelr** [comment]
Oh, If that is the case, coming soon we will allow Indvidual's without legal entities to sign up, since this is one of our most requested features, I'll keep you posted.
